FTB Modpack Downloading Blog/News Post (#262)
PrismLauncher/prismlauncher.org: src/news/ftb-removal.md

---
title: FTB Modpack Downloading Removal.
description: Why FTB Modpack Downloading was Removed.
date: 2023-04-27
tags:
  - News
---

> At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. [Github PR][pr-ftb-removal]

Reviewer: ```suggestion
At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. [GitHub PR][pr-ftb-removal]
```

> Existing instances of FTB modpacks will remain functional, but once the PR is merged, downloading FTB packs directly will no longer be possible.

Reviewer 2: Muggles don't know what merging a PR is. Say "with the next release" instead.

Author: Haha ok I'll do it when I'm home

Reviewer 3: > Muggles don't know what merging a PR is. Say "with the next release" instead.

That feature isn't even developed yet, I think it's too early to say "with the next release"

> Existing instances of FTB modpacks will remain functional, but with the next release, downloading FTB packs directly will no longer be possible.

Reviewer: ```suggestion
Existing instances of FTB modpacks will remain functional, but with the next release, downloading FTB packs within the launcher will no longer be possible.
```

> We will start working on a feature that allows importing FTB Modpacks from the FTB App, as this is acceptable to the FTB team.

Reviewer: ```suggestion
A way to import FTB Modpacks from the FTB App is planned, though no work was done yet. If you think you can help out, [join the development over here][issue-ftb-app-import]!
```

The reason for this request from the FTB team seems to be related to some launchers attempting to monetize FTB Packs.

## Conversations With FTB Team

In the pictures some parts are blurred out due to Privacy concerns.

### Initial Permission Request

<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request1.png" style="width: 40em">
<br/>
<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request2.png" style="width: 40em">
<br/>
<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request3.png" style="width: 40em">

### Request to Remove FTB Modpack Downloading

<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request4.png" style="width: 40em">
<br/>
<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request5.png" style="width: 40em">

### Additional

<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request6.png" style="width: 40em">
<br/>
<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request7.png" style="width: 40em">
<br/>

> <img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request8.png" style="width: 40em">

Reviewer: The alt tags should probably be more descriptive, so people with vision impairments can let the screen reader read out the screenshots.

> ~ PrismLauncher Team

Reviewer: We don't really do this in blog posts :D

> [pr-ftb-removal]: https://github.com/PrismLauncher/PrismLauncher/pull/1040

Reviewer: ```suggestion
[pr-ftb-removal]: https://github.com/PrismLauncher/PrismLauncher/pull/1040
[issue-ftb-app-import]: https://github.com/PrismLauncher/PrismLauncher/issues/235
```

Reviewer 4: @IThundxr are you going to add this link too, or are you leaving it out intentionally?

---
title: FTB Modpack Downloading Removal.
description: Why FTB Modpack Downloading was Removed.
date: 2023-04-27
tags:
  - News
---

> Due to a request from Slowpoke on behalf of the FTB team, we've had to remove the FTB Modpack downloading feature from PrismLauncher. [Github PR][pr-ftb-removal]

Reviewer 3: ```suggestion
Due to a request from Slowpoke on behalf of the FTB team, we've had to remove the FTB Modpack downloading feature from Prism Launcher. [Github PR][pr-ftb-removal]
```

> At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. Github PR

Reviewer 3: Huh?

Reviewer 3: Just remove this line

tags:
  - News
---

Due to a request from Slowpoke on behalf of the FTB team, we've had to remove the FTB Modpack downloading feature from PrismLauncher. [Github PR][Github-PR]

At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. Github PR 

Existing instances of FTB modpacks will remain functional, but once the PR is merged, downloading FTB packs directly will no longer be possible. 

We will start working on a feature that allows importing FTB Modpacks from the FTB App, as this is acceptable to the FTB team. 

The reason for this request from the FTB team seems to be related to some launchers attempting to monetize FTB Packs.

## Conversations With FTB Team

In the pictures some parts are blurred out due to Privacy concerns.

### Initial Permission Request

<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request1.png" style="width: 40em">
<br/>
<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request2.png" style="width: 40em">
<br/>
<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request3.png" style="width: 40em">

> ### Request to Remove FTB  Modpack Downloading

Reviewer: two spaces

<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request4.png" style="width: 40em">
<br/>
<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request5.png" style="width: 40em">

### Additional

<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request6.png" style="width: 40em">
<br/>
<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request7.png" style="width: 40em">
<br/>
<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request8.png" style="width: 40em">

~ PrismLauncher Team

> 

Reviewer: too many newlines. Run markdownlint

> [Github-PR]: https://github.com/PrismLauncher/PrismLauncher/pull/1040

Reviewer: nit: rename to `pr-ftb-removal` or smth like that
